specs/while_loop.yaml

```diff
@@ -0,0 +1,14 @@
+name: "While loop"
+tags: ["shell"]
+command: "while $CONDITION; do\n\t$COMMAND\ndone"
```

Comment:
this is just a direct import from the existing workflows we had checked into the Warp repository, there are still some changes I'm going to make to how we represent identifiers based on @zachlloyd 's feedback, but will do that as a followup

Reply:
I'm going to use `{{`--there's more discussion in https://docs.google.com/document/d/1yyIFf5bdoiAm8IxftNbx-JIGs0nKiL4IWjwIsUsX5Gw/edit about if you're interested!